### PR TITLE
[Docs] Docstring for DeformConv

### PR DESCRIPTION
**Pull request by MeowZheng**

### CODE REVIEW

File: mmcv/ops/deform_conv.py [hunk 1]
```diff
@@ -179,6 +179,27 @@ def _output_size(ctx, input, weight):
 
 
 class DeformConv2d(nn.Module):
+    r"""Deformable 2D convolution
+    Applies a deformable 2D convolution over an input signal composed of
```
Comment (zhouzaida) [inline mmcv/ops/deform_conv.py:183]: blank line

File: mmcv/ops/deform_conv.py [hunk 2]
```diff
@@ -179,6 +179,27 @@ def _output_size(ctx, input, weight):
 
 
 class DeformConv2d(nn.Module):
+    r"""Deformable 2D convolution
```
Comment (zhouzaida) [inline mmcv/ops/deform_conv.py:182]: Deformable 2D convolution.

File: mmcv/ops/deform_conv.py [hunk 3]
```diff
@@ -229,6 +250,17 @@ def reset_parameters(self):
         self.weight.data.uniform_(-stdv, stdv)
 
     def forward(self, x, offset):
+        """Deformable Convolutional forward function
```
Comment (zhouzaida) [inline mmcv/ops/deform_conv.py:253]: Deformable Convolutional forward function.

**Comment by zhouzaida** [comment]
please add the type hints

File: mmcv/ops/deform_conv.py [hunk 4]
```diff
@@ -179,6 +179,28 @@ def _output_size(ctx, input, weight):
...
+        out_channels (int): Number of channels produced by the convolution.
+        kernel_size(int, tuple): Size of the convolving kernel.
+        stride(int, tuple): Stride of the convolution. Default: 1.
+        padding (int or tuple): Zero-padding added to both sides of the input.
+            Default: 0.
+        dilation (int or tuple): Spacing between kernel elements. Default: 1.
+        groups (int): Number of blocked connections from input.
+            channels to output channels. Default: 1.
+        deform_groups (int): Number of deformable group partitions.
+        bias (bool, optional) – If True, adds a learnable bias to the output.
```
Comment (xvjiarui) [inline mmcv/ops/deform_conv.py:200]:
```suggestion
        bias (bool, optional): If True, adds a learnable bias to the output.
```

**Review by xvjiarui — APPROVED** [review]

File: mmcv/ops/deform_conv.py [hunk 5]
```diff
@@ -246,6 +288,19 @@ def forward(self, x, offset):
                       pad_w].contiguous()
         return out
 
+    def __repr__(self):
+        s = self.__class__.__name__
+        s += f'(in_channels={self.in_channels}, '
```
Comment (ZwwWayne) [inline mmcv/ops/deform_conv.py:293]: The repr string is a little bit long here, in one line.  Suggest keeping only one argument in a line.

**Pull request merged**